src/algebra/ordered_ring.lean

```diff
@@ -854,13 +843,13 @@ instance to_nonneg_ring : nonneg_ring α :=
       (ne_of_gt (pos_def.1 pb))⟩,
   ..‹linear_nonneg_ring α› }
 
-@[priority 100] -- see Note [lower instance priority]
-instance to_linear_order : linear_order α :=
+local attribute [instance]
```

Comment:
```suggestion
/-- Construct `linear_order` from `linear_nonneg_ring`. This is not an instance
because we don't use it in `mathlib`. -/
local attribute [instance]
```